automation/check-npm-version.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+'use strict';
+
+/** Check that semver v1 is greater than or equal to semver v2 */
+function semverGte(v1, v2) {
```

Comment:
Yes, reinventing the wheel... The problem with `require('semver')` is that we wanted to run this script in the npm 'preinstall' hook, at which point no dependencies have been installed. As is, this script depends on Node.js only.